src/main/java/com/skhu/luxuryshop/entity/User.java

```diff
@@ -0,0 +1,30 @@
+package com.skhu.luxuryshop.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Getter
+@Builder
+@Entity
+@NoArgsConstructor
```

Comment:
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```

JPA에서 entity에는 기본 생성자가 꼭 필요하죠!
default는 아마.. public인데 많은 분들이 위에 추천 해드린것처럼 
access level을 protected로 제한하는데 왜 그런건지
공부해보시고 access level을 protected로 수정해보아요!